src/org/usfirst/frc4904/robot/subsystems/CrateIO.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import org.usfirst.frc4904.robot.commands.MotorIdleGroup;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * CrateIO is the subsystem of rollers
+ * in the front of the robot, allowing
+ * it to intake/outtake cubes in front
+ * of it.
+ */
+public class CrateIO extends Subsystem {
+
+	public static final double INTAKE_SPEED = 1.0;
+	public static final double OUTTAKE_SPEED = -1.0;
```

Comment:
these should be 0.5

Reply:
at least for now, as that's what we were using today. they may change, but 100% is too much